infrastructure/admission/domain/service/profil_candidat.py

```diff
@@ -603,17 +603,21 @@ def get_experiences_non_academiques(
         experiences_cv_recuperees: ExperiencesCVRecuperees = ExperiencesCVRecuperees.TOUTES,
         uuid_experience: str = '',
     ) -> List[ExperienceNonAcademiqueDTO]:
-        non_academic_experiences: QuerySet[ProfessionalExperience] = ProfessionalExperience.objects.filter(
-            person__global_id=matricule,
-        ).annotate(
-            valuated_from_admissions=ArrayAgg(
-                'valuated_from_admission__uuid',
-                filter=Q(valuated_from_admission__isnull=False),
-            ),
-            injecte_par_admission=Exists(
-                AdmissionEPCInjection.objects.filter(admission__uuid=OuterRef('valuated_from_admission__uuid'))
-            ),
-            injecte_par_cv=Exists(CurriculumEPCInjection.objects.filter(experience_uuid=OuterRef('uuid'))),
+        non_academic_experiences: QuerySet[ProfessionalExperience] = (
+            ProfessionalExperience.objects.filter(
+                person__global_id=matricule,
+            )
+            .annotate(
+                valuated_from_admissions=ArrayAgg(
+                    'valuated_from_admission__uuid',
+                    filter=Q(valuated_from_admission__isnull=False),
+                ),
+                injecte_par_admission=Exists(
+                    AdmissionEPCInjection.objects.filter(admission__uuid=OuterRef('valuated_from_admission__uuid'))
```

Comment:
Je n'y ai pas pensé en patchant urgemment en prod; mais je pense que pour les lignes de CV, il ne faut prendre en compte que les AdmissionEPCInjection de type=DEMANDE

Reply:
C'est noté, j'ai modifié pour prendre en compte ce filtre lors de la récupération, de l'édition et de la suppression des expériences du cv et des études secondaires.